File: tests/fixtures/patterns-positive/strategy_command_functional/mutB.py
# POSITIVO (mutación B: Strategy aplanado en if/elif; cuerpos sustanciales).
class Sorter:
    def sort_by(self, items, mode):
        if mode == "asc":
            return sorted(items)
        elif mode == "desc":
            return sorted(items, reverse=True)
        elif mode == "abs":
            return sorted(items, key=lambda x: abs(x))
        return items

File: tests/fixtures/patterns-positive/strategy_command_functional/mutB.ts
// POSITIVO (mutación B: Strategy aplanado en ifs anidados; cuerpos sustanciales).
class Sorter {
  sortBy(list: number[], mode: "asc" | "desc" | "abs"): number[] {
    if (mode === "asc") {
      return list.slice().sort((a, b) => a - b);
    } else if (mode === "desc") {
      return list.slice().sort((a, b) => b - a);
    } else if (mode === "abs") {
      return list.slice().sort((a, b) => Math.abs(a) - Math.abs(b));
    }
    return list;
  }
}
